CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_reddit_ads v0.3.0
+[PR #13](https://github.com/fivetran/dbt_reddit_ads/pull/13) includes the following updates:
+
```

Comment:
We don't mention the upstream new staging models here at all or reference the other breaking changes in the source package. Can we make sure to include those here.